float64[] transformation_matrix
float64[] xyz_roll_pitch_yaw
float64[] jacobian